caikit/runtime/utils/servicer_util.py

```diff
@@ -26,12 +26,14 @@
 import alog
 
 # Local
+from caikit.core.data_model import DataStream
 from caikit.core.data_model.base import DataBase
 from caikit.core.signature_parsing import CaikitMethodSignature
 from caikit.interfaces.runtime.data_model.training_management import ModelPointer
 from caikit.runtime.model_management.model_manager import ModelManager
 from caikit.runtime.types.caikit_runtime_exception import CaikitRuntimeException
 from caikit.runtime.utils.import_util import get_data_model
+import caikit.core
```

Comment:
Is this pulled in just for `isiterable`? Seems a bit overkill when you could pull it in directly from `caikit.core.toolkit.isa`, but 🤷